internal/services/storagecache/managed_lustre_file_system_resource.go

```diff
@@ -153,6 +161,35 @@ func (r ManagedLustreFileSystemResource) Arguments() map[string]*pluginsdk.Schem
 			},
 		},
 
+		"root_squash_setting": {
```

Comment:
We can drop redundant parts such as `setting` from the property name ([ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md?plain=1#L16))

```suggestion
		"root_squash": {
```